src/components/BB-BUS/IndexFilter.js

```diff
@@ -225,16 +225,12 @@ const styles = StyleSheet.create({
 });
 
 IndexFilter.propTypes = {
-  selectedFilter: PropTypes.object.isRequired,
-  results: PropTypes.array.isRequired,
-  listItems: PropTypes.array.isRequired,
-  setListItems: PropTypes.func.isRequired,
   areaId: PropTypes.string.isRequired,
-  setAreaId: PropTypes.func.isRequired,
   areas: PropTypes.array,
-  loading: PropTypes.bool.isRequired
-};
-
-IndexFilter.defaultProps = {
-  communities: []
```

Comment:
was this before `areas`? and should `areas` prop be an empty array now per default?

Reply:
and areas does not need a default?

Reply:
I don't think we need it because so far we haven't created it for a default data `areas`. So far I haven't encountered any errors. 🤷‍♂️